File: submissions/sm_104_asheeh/week_03/day_4/session1/mirrorImage.js
function mirrorImage(str)
{
 var   arr1 = ['A', 'a', 'B', 'b', 'C', 'c', 'D', 'd', 'E', 'e', 'F', 'f', 'G', 'g', 'H', 'h', 'I', 'i', 'J', 'j', 'K', 'k', 'L', 'l', 'M', 'm'] 
 var   arr2 = ['Z', 'z', 'Y', 'y', 'X', 'x', 'W', 'w', 'V', 'v', 'U', 'u', 'T', 't', 'S', 's', 'R', 'r', 'Q', 'q', 'P', 'p', 'O', 'o', 'N', 'n']
 var   a=''
 for(var i=0 ; i<str.length ; i++)
  {
      for(var j=0 ; j<arr1.length ; j++)
      {
          if(str.charAt(i) == arr1[j])
          {
              a = a + arr2[j]
          }
          else if(str.charAt(i) == arr2[j])
          {
              a = a + arr1[j]
          }
      }
  }
  console.log('The mirror image is:', a)
  return a
}

mirrorImage('AyzB')

File: submissions/sm_009_chinmay/week_13/day_4/part_2/mirror_image.py
# Print the mirror image of the given string AyzB should become ZbaY

lowercase_list = ['a','b','c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z']
uppercase_list = ['A','B','C','D','E','F','G','H','I','J','K','L','M','N','O','P','Q','R','S','T','U','V','W','X','Y','Z']

input_string = 'AyzB'
mirror_string = ''

for i in (input_string):
    for j in range(len(lowercase_list)):
        if lowercase_list[j] == i:
            mirror_string = mirror_string + lowercase_list[25-j]
        elif uppercase_list[j] == i:
            mirror_string = mirror_string + uppercase_list[25-j]
print(mirror_string)